lib/core/src/Cardano/Wallet/Primitive/Types.hs

```diff
@@ -399,6 +401,19 @@ instance Buildable TxStatus where
         InLedger -> "in ledger"
         Invalidated -> "invalidated"
 
+instance FromText TxStatus where
+    fromText txt = case txt of
+        "Pending" -> Right Pending
+        "InLedger" -> Right InLedger
+        "Invalidated" -> Right Invalidated
+        _ ->
+            Left . TextDecodingError $ "not a valid value: " <> show txt
+
+instance ToText TxStatus where
+    toText Pending = "Pending"
+    toText InLedger = "InLedger"
+    toText Invalidated = "Invalidated"
```

Comment:
we've been using lowercase `snake_case` so far to transform constructor to text.

Reply:
done